internal/plugin/service/service/schema.go

```diff
@@ -0,0 +1,204 @@
+			},
+			"service_host": schema.StringAttribute{
+				Description: "The hostname of the service.",
+				Computed:    true,
+			},
+			"service_port": schema.Int64Attribute{
+				Description: "The port of the service",
+				Computed:    true,
+			},
+			"service_password": schema.StringAttribute{
+				Description: "Password used for connecting to the service, if applicable",
+				Computed:    true,
+				Sensitive:   true,
+			},
+			"service_username": schema.StringAttribute{
+				Description: "Username used for connecting to the service, if applicable",
+				Computed:    true,
+			},
+			"state": schema.StringAttribute{
+				Description: "Service state. One of `POWEROFF`, `REBALANCING`, `REBUILDING` or `RUNNING`",
+				Computed:    true,
+			},
+			"static_ips": schema.SetAttribute{
+				Description: "Static IPs that are going to be associated with this service. Please assign a value using the 'toset' function. Once a static ip resource is in the 'assigned' state it cannot be unbound from the node again",
+				ElementType: types.StringType,
+				Optional:    true,
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"components": schema.SetNestedBlock{
+				Description: "Service component information objects",
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"component": schema.StringAttribute{
+							Description: "Service component name",
+							Computed:    true,
+						},
+						"host": schema.StringAttribute{
+							Description: "DNS name for connecting to the service component",
+							Computed:    true,
+						},
+						"kafka_authentication_method": schema.StringAttribute{
+							Description: "Kafka authentication method. This is a value specific to the 'kafka' service component",
+							Computed:    true,
+						},
+						"port": schema.Int64Attribute{
+							Description: "Port number for connecting to the service component",
+							Computed:    true,
+						},
+						"route": schema.StringAttribute{
+							Description: "Network access route",
+							Computed:    true,
+						},
+						"ssl": schema.BoolAttribute{
+							Description: "Whether the endpoint is encrypted or accepts plaintext. By default endpoints are " +
+								"always encrypted and this property is only included for service components they may " +
+								"disable encryption",
+							Computed: true,
+						},
+						"usage": schema.StringAttribute{
+							Description: "DNS usage name",
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"service_integrations": schema.SetNestedBlock{
+				Description: "Service integrations to specify when creating a service. Not applied after initial service creation",
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"source_service_name": schema.StringAttribute{
+							Description: "Name of the source service",
+							Required:    true,
+						},
+						"integration_type": schema.StringAttribute{
```

Comment:
might be worth validating the value of `integration_type` if it only supports `read_replica` for now, consider adding a validation function